mkdocs.yml

```diff
@@ -45,6 +45,16 @@ nav:
     - User Guides:
         - Git as a Data Source: 'user-guides/git-data-source.md'
         - GraphQL: 'user-guides/graphql.md'
+    - Getting Started In the Web UI:
+        - 'user-guides/getting-started/index.md'
+        - Creating Devices: 'user-guides/getting-started/creating-devices.md'
+        - Regions: 'user-guides/getting-started/regions.md'
+        - Platforms: 'user-guides/getting-started/platforms.md'
+        - Tenants: 'user-guides/getting-started/tenants.md'
+        - Interfaces: 'user-guides/getting-started/interfaces.md'
+        - VLANS and VLAN Groups: 'user-guides/getting-started/vlans-and-vlan-groups.md'
+        - The Search Bar: 'user-guides/getting-started/search-bar.md'
```

Comment:
Given that the first example in "The Search Bar" is about searching IP addresses, should this come *after* "IP Address Management"?

Reply:
Yeah, the way it was written before had all of this in a single doc, but when it's broken into sections it reads different.  I changed some wording and made the ordering change you specify